proto/ligato/vpp/dns/models.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 Pantheon.tech
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpp_dns
+
+import "go.ligato.io/vpp-agent/v3/pkg/models"
+
+const (
+	ModuleName = "vpp.dns"
+)
+
+var (
+	// DNSServer is registered NB model of DNSServer
+	ModelDNSServer = models.Register(&DNSServer{}, models.Spec{
+		Module:  ModuleName,
+		Type:    "dnsserver",
```

Comment:
I would maybe go with type "dnscache" (likewise for proto message name), just to make it clear that this is not actually a proper DNS server.
Version can be "v1" since this is the first version of the model.

Reply:
Fixed in [`34ea247`](https://github.com/ligato/vpp-agent/pull/1779/commits/34ea247795b517addafa51056f45b2a4a6d5cba2)